### DISCUSSION TITLE
Enhancement: User-level Usage Logs

### DISCUSSION THREAD

**Discussion by fret423** [danny-avila/LibreChat#4270] [Feature Requests & Suggestions]
### What features would you like to see added?

It would be helpful to have a page (for the admin) where they can see user-level logs similar to the OpenAI Usage page. There are always situations of abuse by a specific user and it is helpful to identify that.

### More details

Even a simple table with usage by user in terms of models and tokens used is a good start

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
> It would be helpful to have a page (for the admin) where they can see user-level logs similar to the OpenAI Usage page. There are always situations of abuse by a specific user and it is helpful to identify that.

If you can identify the userId, you can look through the existing written logs in ./logs directory at project root for a lot of information

> Even a simple table with usage by user in terms of models and tokens used is a good start

This also exists already, check the transactions collection in mongodb
